onnxruntime/core/providers/shared_library/provider_api.h

```diff
@@ -264,7 +264,7 @@ namespace profiling {
   std::string demangle(const char* name);
   std::string demangle(const std::string& name);
 
-};
+} /* namespace profiling */
```

Comment:
nit: I think you should sole rely on clang-format to do the namespace end comments. Otherwise, once a new user format this file, it will cause undesired editing history.